packages/kit/src/utils/filesystem.js

```diff
@@ -105,7 +105,7 @@ export function walk(cwd, dirs = false) {
 		}
 	}
 
-	return (walk_dir(''), all_files);
```

Comment:
Result of pnpm format

Reply:
`lint` on the CI is failing. I suspect this shouldn't have changed. I'm not sure why prettier would be doing something different for you than everyone else. Maybe make sure you're running the latest code from main and do a pnpm install
```
packages/kit lint: [warn] src/utils/filesystem.js
packages/kit lint: [warn] test/apps/basics/src/routes/css/dynamic/+page.svelte
packages/kit lint: [warn] Code style issues found in 2 files. Run Prettier with --write to fix.
```

Reply:
you are correct, I did pull recent changes from upstream, but I forgot to do `pnpm install`. I'll fix it in my other PR as well.